packages/strapi-plugin-content-type-builder/controllers/ContentTypeBuilder.js

```diff
@@ -119,8 +124,8 @@ module.exports = {
       modelJSON.connection = connection;
       modelJSON.collectionName = collectionName;
       modelJSON.info = {
-        name,
-        description
+        name: name,
```

Comment:
We not need {name: name}

```suggestion
        name,
```

works well.